packages/botonic-cli/templates/blank/package.json

```diff
@@ -16,7 +16,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.6.2",
-    "@botonic/react": "0.10.0"
+    "@botonic/react": "~0.10.1"
```

Comment:
I think we can change it to ^0.10.1 to avoid changing it so often. @ericmarcos what do you think?

Reply:
i talked with him and we agreed to leave it as '~' to upgrade only to next patches